.github/restyled.yaml

```diff
@@ -1,6 +1,6 @@
 restylers_version: stable
 restylers:
   - fourmolu:
-      image: 'restyled/restyler-fourmolu:v0.13.0.0'
+      image: 'restyled-io/restyler-fourmolu:v0.18.0.0'
```

Comment:
Not sure why the CI does not pick up the new version. 🤔 
```
Error response from daemon: manifest for restyled/restyler-fourmolu:v0.18.0.0 not found: manifest unknown: manifest unknown
 WARN: Retrying docker-pull
       {
         "attempt": 1,
         "exitCode": 1,
         "limit": 5
       }
```

Based on the restyled repo docs, it was updated in January:
https://github.com/restyled-io/restylers/blob/main/_docs/restylers.md#fourmolu
and there is an image with this name and version:
https://gallery.ecr.aws/restyled-io/restyler-fourmolu